defaults/main.yml

```diff
@@ -1,7 +1,10 @@
+mysql_hardening_enabled: yes
```

Comment:
I'm usually make ability to disable role but not exclude it from playbook. It's very useful, if you have a lot of different builds use one playbook